data/moves.js

```diff
@@ -7143,7 +7147,9 @@ let BattleMovedex = {
 		priority: 0,
 		flags: {},
 		isMax: "Orbeetle",
-		pseudoWeather: 'gravity',
+		self: {
+			pseudoWeather: 'gravity',
```

Comment:
In theory, this one shouldn't be necessary (nor the others).

Reply:
it is necessary, I tested the current code on PS and here are the results: https://replay.pokemonshowdown.com/gen8customgame-1012218343

Reply:
Awkward. I guess this will need to be fixed later.

Reply:
Nice creative hack, though.